test/lib/file.test.js

```diff
@@ -154,8 +154,17 @@ describe('File', function () {
     it('is a getter that returns a string', function () {
       assume(file.location).to.be.a('string');
     });
+    it('returns an external location', function () {
```

Comment:
I'm still a bit unsure about this behaviour. If code is external, wouldn't the developer be in charge of providing the right external location? E.g. including the extension, for instance `bootstrap.min.css`, vs `bootstrap.min`, since its external we have no knowledge of the extension. This is also one of the reasons the future implementation of compiler should have different polymorphed `File` constructors, see #82